en/index.md

```diff
@@ -407,6 +407,51 @@ rather than plain English. Try to keep it simple.
 #21 No double space after a period/full stop
 -->
 
+## Admonitions
+
+Admonitions (also commonly referred to as "admonishments" or sometimes
+"callouts" or "Notifications") are a device used in documentation to draw attention to a
+particular statement or paragraph of text. Typically their use is to highlight:
+
+ - A consequence of performing a particular action
+ - An additional source of information which is useful but not required
+ - A helpful tip that will save effort/time
+ - A reminder of some pre-requisite or restriction
+
+ While this style guide usually concerns itself solely with the intent and
+ structure of content rather than the published form of that content, there is
+ a particular intersect here. The [Vanilla framework][], which is used to
+ publish most of our documentation, only allows for a particular set of
```

Comment:
Should we aim to be more generic for the style guide?

I'm not sure of the exact proportion but a lot of docs are written using Sphinx, which has at least nine different versions of these callout elements. I also think there is four specified in the Vanilla docs.

Suggestion:

> Depending on the documentation framework, a particular set of pre-defined admonitions is made available.
> These admonitions are styled with specific colours and icons.
> For example, a **Warning** admonition will be coloured red and include a warning sign.

Perhaps then an example admonition can be shown.

Reply:
The long term goal of the sphinx project is to have a vanilla theme, not the current one. That will limit us to the choices Vanilla has made. I would rather this section was about when to use or not use an admonition and what it should contain, but I feel like if we suggest the classic set (of 5) we are just setting people up for failure when the theme changes at some point in the future. It kind of makes the distinction between a Warning and a Caution rather irrelevant if you can only have one of them.